sdcm/nemesis.py

```diff
@@ -393,15 +393,16 @@ def get_list_of_methods_by_flags(
         return disrupt_methods_list
 
     def get_list_of_subclasses_by_property_name(self, list_of_properties_to_include):
-        flags = {flag_name: True for flag_name in list_of_properties_to_include}
+        flags = {flag_name.strip('!'): not flag_name.startswith(
+            '!') for flag_name in list_of_properties_to_include}
```

Comment:
If it is a list of `properties` then why calling it `flag_name` and not `property_name`? sounds confusing.

Reply:
it's a property, but since it's a boolean, it's a flag.